nnpdf_data/nnpdf_data/commondata/CMS_Z0_13TEV/metadata.yaml

```diff
@@ -0,0 +1,52 @@
+setname: CMS_Z0_13TEV
+version: 1
+version_comment: 'Initial implementation' 
+arXiv: 
+  url: https://arxiv.org/abs/2202.12327
+  journal: JHEP 2022 (2022) 08, 063
+iNSPIRE:
+  url: https://inspirehep.net/literature/1952936
+hepdata:
+  url: https://www.hepdata.net/record/ins2038801
+  version: 1
+
+nnpdf_metadata:
+  nnpdf31_process: "DY NC"
+  experiment: CMS
+  
+implemented_observables:
+- observable_name: AFB
+  observable:
+    description: "Measurement of the forward backward asymmetry in NC muon and electron pair production at sqrt(s) = 13 TeV"
+    label: "A_{FB}"
+    units: ""
+  process_type: DY_MLL
+  tables: [2]
+  ndata: 7
+  plotting:
+    dataset_label: 'CMS 13 TeV $A_{FB}$ $e$ and $\mu$ channels combined' 
+    plot_x: m_ll
+    y_label: "$A_{FB}$"
+  kinematic_coverage:
+  - m_ll
+  kinematics:
+    variables:
+      m_ll:
+        description: "dilepton mass"
+        label: "$m_{ll}$"
+        units: "$GeV$"
+    file: kinematics_AFB.yaml
+  data_central: data_AFB.yaml
+  data_uncertainties:
+  - uncertainties_AFB.yaml
+  variants:
+     breakdown:
+       data_uncertainties:
+       - uncertainties_breakdown_AFB.yaml
+  theory:
+    FK_tables:
+    - - CMS_Z0_13TEV_FWD
+      - CMS_Z0_13TEV_BWD
```

Comment:
We don't have these grids yet, right?

Do we want to merge it already or wait for the grids? (regardless of whether they are correct or not, just to keep track of the missing ones)